### DISCUSSION TITLE
how to scale contents of just one egui sub-window when multiple are present?

### DISCUSSION THREAD

**Discussion by bcolloran** [emilk/egui#2485] [Q&A] [answered]
I have an app with multiple windows (like the [web demo](https://www.egui.rs/#demo)), and I want to independently set the overall scaling (text, inputs, buttons, all other widgets) for one of them. I've looked around in the api docs but haven't found the right way to do this. Is this possible?

Thanks for any help! :-)

(Background info for context: this is for a game project using bevy. I have an egui-inspector window that is only present in debug builds, and that should use the default scaling to remain small and information dense. I also have game menus that I want to rescale to a constant proportion of the window height when the window is resized or fullscreen mode is entered. I have tried setting the global `ctx.pixels_per_point`, but this makes the content in the egui-inspector too big.)

**Comment by bcolloran** [comment]
Any ideas here? I think this must be doable, I just can't see how. Anyone have a quick answer? Or is it impossible currently with egui?

**Comment by coderedart** [comment, marked as answer]
related https://github.com/emilk/egui/issues/1811

custom scaling of only particular egui containers is not possible atm afaik. 

the best you can do is change the style (text styles in particular) just before running a some particular egui code to have different sized fonts. https://docs.rs/egui/latest/egui/struct.Ui.html#method.style_mut

**Reply by bcolloran** [reply]
Ah ok, bummer. Thanks for getting back to me though!
